libgadget/blackhole.c

```diff
@@ -381,6 +381,9 @@ blackhole(const ActiveParticles * act, ForceTree * tree, FILE * FdBlackHoles, FI
     {
         if(P[i].Type == 0 && priv->Injected_BH_Energy[P[i].PI] > 0)
         {
+            /* Do not heat star-forming particles*/
```

Comment:
This is more like assuming feedback energy is immediately cooled away by the star forming particle.
(Since that we do not inject the BH feedback energy to other particles).

Reply:
This is true! But it would be a bit messy to have the treewalk avoid star forming particles when finding neighbours (and might not converge).